docs/management/direct-k8s-access.md

```diff
@@ -0,0 +1,25 @@
+# Direct Kubernetes Access
+
+Convox makes managing and operating your Kubernetes cluster extremely easy and simple by abstracting a lot of the unnecessary complexities away  We know, however, that there may be times that you will want to dive deeper and access the k8s primitives and resources yourself.
+
+This could be deep debugging of an edge-case networking issue, examination of an underlying cloud infrastructure issue, gaining a deeper understanding of how everything ties together, or simply to install a Kubernetes native component or plugin directly.
+
+Convox provides an API proxy to Kubernetes that runs on your Rack which allows you to grant access to the underlying system, whilst still respecting and adhering to the Role-Based Access Control (RBAC) that Kubernetes provides.  This is actually a lot easier and more manageable than providing direct Kubernetes credentials to the developers in your team!
```

Comment:
We don't really do anything with Kubernetes RBAC. The main advantage is because of this proxy if you're using Console you can delegate per-user access to Kubernetes.